backend/tests/unit/config/test_settings.py

```
@@ -352,8 +352,8 @@ def test_builtin_providers_config_structure(self):
         """Test that built-in provider configs have all required fields."""
         from tarsy.config.builtin_config import BUILTIN_LLM_PROVIDERS
         
-        required_fields = {"type", "model", "api_key_env", "temperature"}
-        optional_fields = {"base_url", "verify_ssl", "max_tool_result_tokens"}
+        required_fields = {"type", "model", "api_key_env"}
+        optional_fields = {"base_url", "verify_ssl", "max_tool_result_tokens", "temperature"}
```

Ruff (0.14.5)

Local variable `optional_fields` is assigned to but never used

Remove assignment to unused variable `optional_fields`

(F841)